jazzhands/__main__.py

```diff
@@ -219,22 +236,32 @@ def main(argv):
 
         while 1:
             changed = False
```

Comment:
This looks pretty inefficient. Maybe sometime soon we could switch to something like https://pypi.org/project/watchdog/?

Meanwhile, can we add a brief sleep in each loop so we're not running a CPU core all-out just checking for file changes?

Reply:
👍